src/push/request.ts

```diff
@@ -34,7 +34,6 @@ const { version } = require('../../package.json');
 
 export type APISchema = {
   project: string;
-  keep_stale: boolean;
```

Comment:
We would need this flag if i remember correctly from my conversation with Dominique.

Reply:
@shahzad31 actually decided to delete that requirement from the Kibana side as well.